apps/studio/src/lib/db/clients/sqlite.js

```diff
@@ -38,6 +38,7 @@ export default async function (server, database) {
     supportedFeatures: () => ({ customRoutines: false, comments: false, properties: true }),
     versionString: () => getVersionString(version),
     wrapIdentifier,
+    defaultSchema: () => '',
```

Comment:
@wmontgomery i think if there is no schema this should return null, no?  Aren’t we going to want to do checks like if(schema).

Reply:
This is how we handle no schema elsewhere in the code. Can we make this change and check table creation still works?

Reply:
I'm changing it to return null